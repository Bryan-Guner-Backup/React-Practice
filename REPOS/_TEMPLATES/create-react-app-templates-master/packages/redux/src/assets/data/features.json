[
  {
    "label": "Redux",
    "description": "Redux is a predictable state container for JavaScript apps",
    "website": "http://redux.js.org/"
  },
  {
    "label": "Jest",
    "description":
      "Jest is used by Facebook to test all JavaScript code including React applications.",
    "website": "https://facebook.github.io/jest/"
  },
  {
    "label": "Enzyme",
    "description":
      "Enzyme is a JavaScript Testing utility for React that makes it easier to assert, manipulate, and traverse your React Components' output.",
    "website": "http://airbnb.io/enzyme/"
  },
  {
    "label": "React Router",
    "description": "Declarative routing for React",
    "website": "https://reacttraining.com/react-router/"
  },
  {
    "label": "react-router-redux",
    "description":
      "Ruthlessly simple bindings to keep react-router and redux in sync",
    "website":
      "https://github.com/ReactTraining/react-router/tree/master/packages/react-router-redux"
  },
  {
    "label": "react-redux-modules",
    "description":
      "React component modules that easily integrate with your Redux store",
    "website": ""
  },
  {
    "label": "React-MD",
    "description":
      "react-md is a set of React components and sass files for implementing Google's Material Design.",
    "website": "https://react-md.mlaursen.com/"
  },
  {
    "label": "material-design-icons",
    "description":
      "Material icons are beautifully crafted, delightful, and easy to use in your web, Android, and iOS projects.",
    "website": "https://material.io/icons/"
  },
  {
    "label": "node-sass-chokidar",
    "description":
      "A thin wrapper around node-sass to replicate the --watch using chokidar instead of Gaze.",
    "website": "https://github.com/michaelwayman/node-sass-chokidar"
  },
  {
    "label": "Prettier",
    "description":
      "JavaScript formatter inspired by refmt with advanced support for language features from ES2017, JSX, and Flow.",
    "website": "https://github.com/prettier/prettier/"
  }
]
