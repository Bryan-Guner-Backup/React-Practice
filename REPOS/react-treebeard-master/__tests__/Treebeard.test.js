import React from 'react';
import {shallow} from 'enzyme';

import Treebeard from '../src/components';
import data from './mocks/data';

const renderComponent = (props = {}) => {
    const wrapper = shallow(
        <Treebeard data={props.data || data} {...props}/>
    );
    wrapper.treeNode = () => wrapper.find('TreeNode');
    return wrapper;
};

describe('<Treebeard/>', () => {
    it('should match default snapshot', () => {
        const wrapper = renderComponent();
        expect(wrapper).toMatchSnapshot();
    });
});
