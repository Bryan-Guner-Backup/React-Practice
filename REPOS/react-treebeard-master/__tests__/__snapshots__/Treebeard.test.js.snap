// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Treebeard/> should match default snapshot 1`] = `
<Styled(ul)
  style={
    Object {
      "backgroundColor": "#21252B",
      "color": "#9DA5AB",
      "fontFamily": "lucida grande ,tahoma,verdana,arial,sans-serif",
      "fontSize": "14px",
      "listStyle": "none",
      "margin": 0,
      "padding": 0,
    }
  }
>
  <TreeNode
    animations={
      Object {
        "drawer": [Function],
        "toggle": [Function],
      }
    }
    customStyles={Object {}}
    decorators={
      Object {
        "Container": [Function],
        "Header": [Function],
        "Loading": Object {
          "$$typeof": Symbol(react.forward_ref),
          "__emotion_base": [Function],
          "__emotion_forwardProp": undefined,
          "__emotion_real": [Circular],
          "__emotion_styles": Array [
            [Function],
          ],
          "defaultProps": undefined,
          "displayName": "Styled(Component)",
          "propTypes": Object {
            "style": [Function],
          },
          "render": [Function],
          "withComponent": [Function],
        },
        "Toggle": [Function],
      }
    }
    key="1"
    node={
      Object {
        "children": Array [
          Object {
            "children": Array [
              Object {
                "name": "app.js",
              },
              Object {
                "name": "data.js",
              },
              Object {
                "name": "index.html",
              },
              Object {
                "name": "styles.js",
              },
              Object {
                "name": "webpack.config.js",
              },
            ],
            "name": "example",
          },
          Object {
            "children": Array [],
            "loading": true,
            "name": "node_modules",
          },
          Object {
            "children": Array [
              Object {
                "children": Array [
                  Object {
                    "name": "decorators.js",
                  },
                  Object {
                    "name": "treebeard.js",
                  },
                ],
                "name": "components",
              },
              Object {
                "name": "index.js",
              },
            ],
            "name": "src",
          },
          Object {
            "children": Array [
              Object {
                "name": "animations.js",
              },
              Object {
                "name": "default.js",
              },
            ],
            "name": "themes",
          },
          Object {
            "name": "gulpfile.js",
          },
          Object {
            "name": "index.js",
          },
          Object {
            "name": "package.json",
          },
        ],
        "id": 1,
        "name": "react-treebeard",
        "toggled": true,
      }
    }
    style={
      Object {
        "activeLink": Object {
          "background": "#31363F",
        },
        "base": Object {
          "position": "relative",
        },
        "header": Object {
          "base": Object {
            "color": "#9DA5AB",
            "display": "inline-block",
            "verticalAlign": "top",
          },
          "connector": Object {
            "borderBottom": "solid 2px black",
            "borderLeft": "solid 2px black",
            "height": "12px",
            "left": "-21px",
            "position": "absolute",
            "top": "0px",
            "width": "2px",
          },
          "title": Object {
            "lineHeight": "24px",
            "verticalAlign": "middle",
          },
        },
        "link": Object {
          "cursor": "pointer",
          "display": "block",
          "padding": "0px 5px",
          "position": "relative",
        },
        "loading": Object {
          "color": "#E2C089",
        },
        "subtree": Object {
          "listStyle": "none",
          "paddingLeft": "19px",
        },
        "toggle": Object {
          "arrow": Object {
            "fill": "#9DA5AB",
            "strokeWidth": 0,
          },
          "base": Object {
            "display": "inline-block",
            "height": "24px",
            "marginLeft": "-5px",
            "position": "relative",
            "verticalAlign": "top",
            "width": "24px",
          },
          "height": 14,
          "width": 14,
          "wrapper": Object {
            "height": "14px",
            "left": "50%",
            "margin": "-7px 0 0 -7px",
            "position": "absolute",
            "top": "50%",
          },
        },
      }
    }
  />
</Styled(ul)>
`;
